python/dgl/graphbolt/subgraph_sampler.py

```diff
@@ -131,11 +131,14 @@ def _seeds_preprocess(minibatch):
                         else None
                     )
                     return seeds, nodes_timestamp, None
-                assert typed_seeds.ndim == 2 and typed_seeds.shape[1] == 2, (
-                    "Only tensor with shape 1*N and N*2 is "
+                assert typed_seeds.ndim == 2, (
```

Comment:
Can you also update https://docs.dgl.ai/en/latest/generated/dgl.graphbolt.ItemSetDict.html? add an case for hyperlink.